rust/Cargo.toml

```diff
@@ -11,6 +11,9 @@ crate-type = ["staticlib"] # Creates static lib
 
 [dependencies]
 move-core-types = { git = "https://github.com/move-language/move", rev = "f7137eabc2046f76fdad3ded2c51e03a3b1fbd01", features = ["address32"] }
+starknet-crypto = "0.1.0"
```

Comment:
My only concern left is this: let's revisit this when app decides to integrate IMX
> Note that starknet-rs is still experimental. Breaking changes will be made before the first stable release. The library is also NOT audited or reviewed for security at the moment. Use at your own risk.